.circleci/config.yml

```
@@ -268,6 +271,123 @@ jobs:
+      - persist_to_workspace:
+          root: .
+          paths:
+            - Gemfile
+            - Gemfile.lock
+            - fastlane/
+  promo-screenshots:
+    parameters:
+      post-to-slack:
+        description: Post to Slack when screenshots job has finished. SLACK_WEBHOOK ENV variable must be set.
+        type: boolean
+        default: true
+    macos:
+      xcode: 11.6.0
+    working_directory: /tmp/workspace
+    steps:
+      - attach_workspace:
+          at: .
+      - restore_cache:
+          # ImageMagick is used by the 'rmagick' gem, so cache invalidation of ImageMagick should depend on version of rmagick from Gemfile.lock
+          keys:
+            - &homebrew-cache-key v2-brew-imagemagick-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Brew Install ImageMagick
+          command: |
+            brew install pkg-config imagemagick
+            brew cleanup
+      - save_cache:
+          paths:
+            - ~/Library/Caches/Homebrew
+          key: *homebrew-cache-key
+      - run:
+          name: Bundle install with rmagick
+          command: |
+            # Activate 'screenshots' in bundle config options (remove it from 'without' in case it's set here)
+            bundle config --local --delete without
+            bundle config --local with screenshots
+            bundle install --path vendor/bundle
+      - run:
+          name: Download Promo Strings
+          command: bundle exec fastlane download_promo_strings
+      - run:
+          name: Create Promo Screenshots
+          command: bundle exec fastlane android create_promo_screenshots force:true
+      - run:
+          name: ZIP Metadata
+          command: cd fastlane/metadata; zip -r Android-Promo-Screenshots.zip android
+      - store_artifacts:
+          path: fastlane/metadata/Android-Promo-Screenshots.zip
+      - when:
+          condition: << parameters.post-to-slack >>
+          steps:
+            - slack/status:
+                include_job_number_field: false
+                include_project_field: false
+                webhook: '${SLACK_BUILD_WEBHOOK}'
+                # [Trial] Validate URLs (and IDs used within those) in both those messages and adapt to the ones for a8c instances once we integrate the PR from fork to main repo
+                failure_message: ':red_circle: WordPress Android Screenshots failed on \`${CIRCLE_BRANCH}\` branch. See <https://console.firebase.google.com/u/0/project/api-project-108380595987/testlab/histories/bh.e0c3a59bd9ed670|Firebase console test results> for details.\n\nPlease reach out in #platform9 if you think this failure is not caused by your changes, so we can investigate.'
+                success_message: ':tada: WordPress Android Promo Screenshots have been generated and are ready to be reviewed. Download them <https://${CIRCLE_BUILD_NUM}-9306568-gh.circle-artifacts.com/0/fastlane/metadata/Android-Promo-Screenshots.zip>|here>'
```

Just noticed this small typo in my original commit:
```suggestion
                success_message: ':tada: WordPress Android Promo Screenshots have been generated and are ready to be reviewed. Download them <https://${CIRCLE_BUILD_NUM}-9306568-gh.circle-artifacts.com/0/fastlane/metadata/Android-Promo-Screenshots.zip|here>'
```